rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/NodeSource.java

```diff
@@ -777,8 +776,8 @@ public void finishNodeSourceShutdown(Client initiator) {
      * @return a list of alive nodes
      */
     @ImmediateService
-    public LinkedList<Node> getAliveNodes() {
-        LinkedList<Node> nodes = new LinkedList<>();
+    public LinkedList<RMNode> getAliveNodes() {
+        LinkedList<RMNode> nodes = new LinkedList<>();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "nodes" which hides the field declared at line 141. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)